doozerlib/runtime.py

```diff
@@ -593,6 +593,15 @@ def filter_disabled(n, d):
                 self.image_name_map[name] = img.key
                 self.image_name_map[short_name] = img.key
 
+                name_in_bundle = img.data.get('name_in_bundle', None)
+                if name_in_bundle and name_in_bundle!=name and name_in_bundle!=name:
+                    if name_in_bundle in self.image_name_map:
+                        self.logger.warning(
+                            f"name_in_bundle field in conflict with another image name/shortname, for image={name} "
+                            f"name_in_bundle={name_in_bundle}.")
+                    else:
+                        self.image_name_map[name_in_bundle] = img.key
```

Comment:
It's ok but I'd love to use a separate map for `name_in_bundle`s because they are separate concepts.